Source/CheckoutTheme.swift

```diff
@@ -1,9 +1,9 @@
 import UIKit
 
 /// Class used to customize the views.
-public final class CheckoutTheme {
+public enum CheckoutTheme {
```

Comment:
This is great change 👍 and why we should make sure to use linting.

Weirdly though this file shouldn't exist anymore but must've gotten lost in the conflicts from recent PRs, will make a note of it :)